flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/StringWriter.java

```diff
@@ -54,13 +57,15 @@ public StringWriter() {
 	 *
 	 * @param charsetName Name of the charset to be used, must be valid input for {@code Charset.forName(charsetName)}
 	 */
-	public StringWriter(String charsetName) {
+	public StringWriter(String charsetName,byte[] rowDelimiter ) {
```

Comment:
There should exist a space before `byte[] rowDelimiter`

Reply:
If we have already provided the `charsetName`, why not just provided the `String` `delimiter` as parameter and the use `delimiter.getBytes(charsetName)` to get the target bytes.